test/test_uploader.py

```diff
@@ -1076,6 +1076,11 @@ def test_download_multi(self):
         self.assertIn("timestamp", parameters)
         self.assertIn("signature", parameters)
 
+    @unittest.skipUnless(cloudinary.config().api_secret, "requires api_key/api_secret")
+    def test_create_zip_with_target_asset_folder(self):
```

Comment:
@aleksandar-cloudinary, can you please rewrite this test using mock? or add UNIQUE_TAG that will cause the generated zip to be deleted after the test.
Currently the file is left of the cloud after running the test